app/all-recipes.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>&#129472; Good Food</title>
+    <link href="https://fonts.googleapis.com/css?family=Bungee|Roboto+Slab:300,400,700" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" type="text/css">
+  </head>
+  <body>
+    <h1 class="title-all">All the recipes you could ever want</h1>
+
+    <section class="recipe-list">
+      <div class="recipe-card">
```

Comment:
Let's add an anchor tag so we remember to be able to route to full recipe on click.